tools/sync_audit.py

```
@@ -0,0 +1,194 @@
+    result = subprocess.run(
+        [GIT_BIN, "rev-parse", "--abbrev-ref", "HEAD"],
+        capture_output=True,
+        text=True,
+        cwd=worktree,
+    )
+    return result.stdout.strip() if result.returncode == 0 else ""
+
+
+def git_log_subjects(worktree: Path) -> list[str]:
+    """Return all commit subjects (no PR refs) for the worktree's current branch."""
+    result = subprocess.run(
```

Ruff (0.15.12)

[error] `subprocess` call: check for execution of untrusted input

(S603)